src/createStore.js

```diff
@@ -125,6 +127,11 @@ export default function createStore(reducer, initialState) {
     dispatch({ type: ActionTypes.INIT });
   }
 
+  /**
```

Comment:
This is not really JSDoc, is it? It's describing a line of code, not a variable or export. Should it have `/**`?